docs/FAQ/FAQ.md

```diff
@@ -20,12 +20,14 @@ Here are some commonly-asked questions and their answers. Another source of info
 
     ***Parallelization:***
-    - The *Classic Driver* code is not explicitly written for parallel processing. Nevertheless, its design will allow you to implement a "poor man's parallelization". Because VIC simulates each grid cell independently of the others, you can break up your study domain into separate sub-domains (these could be sub-basins or simply any arbitrary grouping of grid cells) and simulate each of these sub-domains separately, in parallel (on different nodes of a cluster, for example). Dividing the domain into sub-domains and combining them again later should be fairly easy, since VIC's input forcings and output results are organized into one file per grid cell. All you need to do is create separate soil parameter files, one for each sub-domain (these can even be copies of the original whole-domain soil parameter file, with the appropriate grid cells given a "1" as the first value and all other grid cells given a "0"). You will also need to create separate global parameter files. These would all be identical save for the soil parameter file that they point to. Of course the routing model must access all grid cells in a single run, but it runs very quickly in comparison to the VIC run.
-    - The *Image* and *CESM Drivers* utilize a the MPI standard for parallelization. This allows VIC to be efficiently run on larger clusters and super-computers.
+
+    - The *[Classic Driver](../Documentation/Drivers/Classic/ClassicDriver.md)* code is not explicitly written for parallel processing. Nevertheless, its design will allow you to implement a "poor man's parallelization". Because VIC simulates each grid cell independently of the others, you can break up your study domain into separate sub-domains (these could be sub-basins or simply any arbitrary grouping of grid cells) and simulate each of these sub-domains separately, in parallel (on different nodes of a cluster, for example). Dividing the domain into sub-domains and combining them again later should be fairly easy, since VIC's input forcings and output results are organized into one file per grid cell. All you need to do is create separate soil parameter files, one for each sub-domain (these can even be copies of the original whole-domain soil parameter file, with the appropriate grid cells given a "1" as the first value and all other grid cells given a "0"). You will also need to create separate global parameter files. These would all be identical save for the soil parameter file that they point to. Of course the routing model must access all grid cells in a single run, but it runs very quickly in comparison to the VIC run.
+    - The *[Image](../Documentation/Drivers/Image/ImageDriver.md) and [CESM](../Documentation/Drivers/CESM/CESMDriver.md) Drivers* utilize a the MPI standard for parallelization. This allows VIC to be efficiently run on larger clusters and super-computers.
```

Comment:
Maybe point to the image driver "how to run VIC" page? https://github.com/jhamman/VIC/blob/c88152065e24e54d44861e70ae0de731e9dda9aa/docs/Documentation/Drivers/Image/RunVIC.md